packages/interaction-kit/src/interactions/index.ts

```diff
@@ -15,54 +15,42 @@ import { ApplicationCommandInteraction, PingInteraction } from "..";
 import SlashCommandAutocompleteInteraction from "./autocomplete/application-command-autocomplete";
 import {
 	APIApplicationCommandInteraction,
-	APIChatInputApplicationCommandInteraction,
-	APIContextMenuInteraction,
 	APIInteraction,
 	APIMessageComponentInteraction,
-	ApplicationCommandType,
-	ComponentType,
 	InteractionType,
-	APIMessageComponentButtonInteraction,
-	APIMessageComponentSelectMenuInteraction,
-} from "discord-api-types/v9";
-import {
-	isAutocompleteExecutableOption,
-	isAutocompleteOption,
-} from "../commands/options/option";
-
-function isChatInputApplicationCommandInteraction(
-	interaction: APIApplicationCommandInteraction
-): interaction is APIChatInputApplicationCommandInteraction {
-	return interaction.data.type === ApplicationCommandType.ChatInput;
-}
+	Utils,
+} from "discord-api-types/v10";
+import Option, { isAutocompleteOption } from "../commands/options/option";
+import { StringOption, NumberOption, IntegerOption } from "../commands/options";
+
+// TODO: Ask Vlad if we can get a version of discord-api-types that doesn't nest this under Utils so we can import it natively with modules
```

Comment:
I am split on this. I'll probably clean up utility functions someday and export at the top level but no promises for now 🙏

Reply:
Fair enough!